src/styles/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: $card-width;
+  border: $card-border;
+  border-radius: 5px;
+  box-sizing: border-box;
+  cursor: pointer;
+  transition-duration: $transition-duration;
```

Comment:
Ensure that the variable `$transition-duration` is defined in your SCSS files. If it's not defined, this will cause a compilation error.